docs/topics/generics.md

```diff
@@ -36,12 +39,12 @@ compiled but caused an exception at runtime:
 ```java
 // Java
 List<String> strs = new ArrayList<String>();
-List<Object> objs = strs; // !!! A compile-time error here saves us from a runtime exception later.
-objs.add(1); // Put an Integer into a list of Strings
-String s = strs.get(0); // !!! ClassCastException: Cannot cast Integer to String
+List<Object> objs = strs; // Java reports a type mismatch here at compile-time, but what if it didn't?
+objs.add(1); // We would be able to put an Integer into a list of Strings
+String s = strs.get(0); // And then at runtime, Java would throw a ClassCastException: Integer cannot be cast to String
```

Comment:
Can you wrap your code comments so that the reader doesn't have to scroll to see the end of the comment?